spring-cloud-kubernetes-client-config/pom.xml

```diff
@@ -74,6 +74,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
I've added a `KubernetesClientBootstrapConfigurationTests` test and need as such this dependency. We had the same test in fabric8 implementation (that used to fail during my development of this PR), and I have realized we do not have it in k8s-native client.